tools/StandardAnchorTags/GenerateGrammar.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StandardAnchorTags
```

Comment:
As we're adding a new file, maybe let's start with a file-scoped namespace? (I'm generally reluctant to retrofit as it messes up git blame, but I usually use it for new code.)

Reply:
I also plan to make an update once .NET 8 ships to make one PR that updates the code to C# 12. (List patterns in particular would be useful.)